### DISCUSSION TITLE
安卓导出的dtb怎么适配armbian

### DISCUSSION THREAD

**Discussion by ColtPlus** [ophub/amlogic-s9xxx-armbian#2388] [Rockchip]
我有一块rk3568b2的板子，目前只能用bpi-r2p启动系统，我从板子固件和系统中都导出了dtb文件，我转成dts后该怎么修改，有模板或者教程吗，路过的大佬指指路，我下一步该怎么做

**Comment by lzw20201111** [comment]
不通用。不过有参数可以参考。。。
